### PR TITLE
only split on first namespace =

### PR DESCRIPTION
**Pull request by anneeb**

### CODE REVIEW

**Comment by brianru** [comment]
Would this work with the example in the bug report? In that example the last `=` is the end of the namespace. https://github.com/stardog-union/stardog-studio/issues/1245

**Comment by anneeb** [comment]
@brianru it should

```
> const str = '1=2=3=4=5='
undefined
> const [a,...b]=str.split('=')
undefined
> b
[ '2', '3', '4', '5', '' ]
> b.join('=');
'2=3=4=5='
```

EDIT:  it also works for multiple trails
```
> const str2 = '1=2=3=4=5===='
undefined
> const [c,...d]=str2.split('=')
undefined
> d
[ '2', '3', '4', '5', '', '', '', '' ]
> d.join('=')
'2=3=4=5===='
```

File: packages/stardog-language-utils/src/__tests__/namespaceUtils.test.ts [hunk 1]
```diff
@@ -1,10 +1,26 @@
...
+  splitNamespace,
   abbreviatePrefixArray,
   namespaceObjToArray,
   namespaceArrayToObj,
   abbreviatePrefixObj,
 } from '../namespaceUtils';
 
+describe('splitNamespace', () => {
+  it('splits a namespace into a string array of the alias and prefix', () => {
+    const namespace = 'foo=http://foo.org/';
```
Comment (brianru) [inline packages/stardog-language-utils/src/__tests__/namespaceUtils.test.ts:11]: heh, thanks. i initially forgot these look like `<abbreviation>=<namespace iri>`

File: packages/stardog-language-utils/src/namespaceUtils.ts [hunk 2]
```diff
@@ -21,6 +21,11 @@ const abbreviate = (
   return newIri;
 };
 
+export const splitNamespace = (namespace: string) => {
+  const [alias, ...splitPrefix] = namespace.split('=');
```
Comment (brianru) [inline packages/stardog-language-utils/src/namespaceUtils.ts:25]: simpler to use the `limit` 2nd param to `split`?

Comment (anneeb) [reply packages/stardog-language-utils/src/namespaceUtils.ts:25]: limit does not return the leftover text

Comment (brianru) [reply packages/stardog-language-utils/src/namespaceUtils.ts:25]: weird

**Review by brianru — COMMENTED** [review]
🐒

**Review by brianru — APPROVED** [review]
🐒

File: packages/stardog-language-utils/src/namespaceUtils.ts (hunk 2 above)
Comment (anneeb) [reply packages/stardog-language-utils/src/namespaceUtils.ts:25]:
```
'1=2=3='.split('=',2)
>[ '1', '2' ]
```

**Pull request merged**